bcManager/__init__.py

```diff
@@ -1,5 +1,5 @@
 
-from .config import config
+from .bcConfig import bcConfig
```

Comment:
Tiny nitpick, but the naming convention of the module has capital `BC` directly below this, and you went with lower case `bc`. Feels incorrect. Obviously it doesn't matter in code, but I love consistency.